easybuild/tools/options.py

```diff
@@ -444,6 +446,12 @@ def postprocess(self):
             if token is None:
                 raise EasyBuildError("Failed to obtain required GitHub token for user '%s'", self.options.github_user)
 
+        # parse file specifying metadata for external modules
+        if self.options.external_modules_metadata:
+            self.options.external_modules_metadata = ConfigObj(self.options.external_modules_metadata)
```

Comment:
i don't know what error to expect here with incalid syntax, but catch them here and reraise them with proper message saying they come from external_modules_metadata